libgadget/density.c

```diff
@@ -539,6 +538,11 @@ density_postprocess(int i, TreeWalk * tw)
 
         SPHP(i).DivVel /= SPHP(i).Density;
         P[i].DtHsml = (1.0 / NUMDIMS) * SPHP(i).DivVel * P[i].Hsml;
+
+        MyFloat * gradrho = DENSITY_GET_PRIV(tw)->GradRho;
+        /* Before this is |grad rho|^2, after it is |grad rho|*/
```

Comment:
Perhaps note this near the declaration of gradrho too.

Reply:
Done!